converter/converter/cisu_converter.py

```diff
@@ -45,51 +45,74 @@ class CISUConverterV3:
     def from_cisu(cls, input_json: Dict[str, Any]) -> Dict[str, Any]:
         """
         Convert from CISU to Health format
-        
+
         Args:
             input_json: Input EDXL CISU JSON
-            
+
         Returns:
             Converted EDXL Health JSON
         """
+
         # Create independent envelope copy without usecase for output
         output_json = copy.deepcopy(input_json)
         if 'createCase' not in input_json.get('content', [{}])[0].get('jsonContent', {}).get('embeddedJsonContent', {}).get('message', {}):
             raise ValueError("Input JSON must contain 'createCase' key")
         del output_json['content'][0]['jsonContent']['embeddedJsonContent']['message']['createCase']
-            
+
         # Create independent usecase copy for output
         input_usecase_json = input_json['content'][0]['jsonContent']['embeddedJsonContent']['message']['createCase']
         output_usecase_json = copy.deepcopy(input_usecase_json)
 
-        # - Deletions
-        delete_paths(output_usecase_json, cls.CISU_PATHS_TO_DELETE)
-        
         # - Updates
         # Set owner to target recipient
         output_usecase_json['owner'] = get_recipient(input_json)
-            
+
         # Handle victims information
-        if input_usecase_json.get('qualification', {}).get('victims') and input_usecase_json.get('initial_alert'):            
-            if 'notes' not in output_usecase_json['initialAlert']:
-                output_usecase_json['initialAlert']['notes'] = []
-                
+        if input_usecase_json.get('qualification', {}).get('victims'):
             victims_text = format_object(input_usecase_json['qualification']['victims'])
-            output_usecase_json['initialAlert']['notes'].append({"freetext": victims_text})
+            add_object_to_initial_alert_notes(output_usecase_json, victims_text)
+
+        # Location
+        if input_usecase_json.get('location', {}).get('city', {}).get('detail'):
+            output_usecase_json['location']['freetext']=input_usecase_json['location']['city']['detail']
+
+        # Qualification - case detail priority
+        if input_usecase_json.get('initialAlert', {}).get('reporting'):
+            if 'details' not in output_usecase_json['qualification']:
```

Comment:
J'avais pas fait ça très proprement mais je pense qu'on peut profiter de la stabilisation pour mieux checker tous les chemins (y compris quand le modèle est en 1..1 sur le champ comme qualification ici) pour éviter les problèmes et les checker de façon cohérente avec tous des get('xxx', {}) plutôt que des not in aaa['xxx']. Tu peux aussi faire un helper checkExists qui prend un jsonpath et check si $.initialAlert.reporting existe si tu trouves que ça rend plus lisible et uniforme